src/app.js

```diff
@@ -37,11 +38,7 @@ const thursdayMeals = function(dayOfSchedule) {
   const week = parseInt(dayOfSchedule / 7);
 
   if (week == 2) {
```

Comment:
Do we want it for weeks 3 and 4 as well as 2? Same for the other `if`, weeks 2 and 3 as well?